js/jquery.js

```diff
@@ -0,0 +1,12 @@
+$(document).ready(function() {
+
+  function() {
```

Comment:
Don't forget to name your function. Down in your click handler you call a function called `showNav` but this function doesn't have a name. This line should look like `function showNav() {`